compiler-js/src/test/ceylon/dyns/issues_dyns.ceylon

```diff
@@ -330,3 +331,9 @@ void issue5959() {
     check((d of Object) is C5959<String>, "#5959 targs3");
   }
 }
+
+void issue6057() {
+  dynamic DefaultObject {}
+  dynamic { DefaultObject o = eval("this"); }
+  check(identityHash(object extends Basic() {}) > 0, "#6057 1");
```

Comment:
This is only true if `identityHash` has been called before, but we don't know if it has. If it's the first time it's ever called, then it returns 0.